.scripts/process_build.sh

```diff
@@ -0,0 +1,161 @@
+#!/usr/bin/env bash
+
+# This script is called from everest-ci/ci script for a weekly build of the FStar Binaries
+# If ran separately, the starting directory should be the root directory of FStar.
+
+# Sorry, everyone
+if (( ${BASH_VERSION%%.*} < 4 )); then
+  echo "This script requires Bash >= 4. On OSX, try: brew install bash"
+  exit 1
+fi
+
+# Any error is fatal.
+set -e
+set -o pipefail
+
+# Expects to be called from $BN_BINARYSPATH_ROOT
+function cp_to_binaries () {
+  local file=$1
+  echo "--" $ORIG_PWD/src/ocaml-output/$file $BN_BINARYSPATH
+  cp $ORIG_PWD/src/ocaml-output/$file $BN_BINARYSPATH
+  git add $BN_BINARYSPATH/$file
+}
+
+function cleanup_files () {
+  local suffix=$1
+  local files=$BN_BINARYSPATH/*.$suffix
+  local file_count=$(ls -1 $files 2>/dev/null | wc -l)
+  if (( $file_count > $BN_FILESTOKEEP )); then
+    # Windows git rm just needs the file name and fails if give path so just get file name
+    local file_list=$(ls -t1 $files | xargs -n1 basename | tail -n +$(($BN_FILESTOKEEP+1)))
+    git rm ${file_list} -f
+  fi
+}
+
+BUILD_DIR=$(pwd)
+if [[ -f ~/.bash_profile ]]; then
+  source ~/.bash_profile
+fi
+cd "$BUILD_DIR"
```

Comment:
do we still need this knowing that the script is called via the `ci` script, which already does this? also, BUILD_DIR is redundant with ORIG_PWD, which briefly got me confused.